### DISCUSSION TITLE
Crossclaim Piston

### DISCUSSION THREAD

**Discussion by anonisnap** [GriefPrevention/GriefPrevention#1057] [Ideas]
Allowing pistons to push across claims made by the same user

To help set an example;
I have made a claim out of a bunch of squares of different sizes because a rectangular claim would not fit, and these claims are all owned by me and stacked right up against each other. I would like to have pistons push a flying machine across some of these claims, but when a claim stops it just stops working at all.

**Comment by Jikoo** [comment]
Set `GriefPrevention.LimitPistonsToLandClaims` to false to allow pistons to push across claim boundaries. This allows pistons to push between claims with the same owner.

**Comment by RoboMWM** [comment]
Meanwhile I'm here wishing that flying machines weren't possible lol.
